saleor/payment/tests/test_payment.py

```diff
@@ -80,6 +80,26 @@ def transaction_data(payment_dummy, gateway_response):
     }
 
 
+@pytest.fixture
+def transaction_data_long_error_message(payment_dummy):
+    error_msg = (
+        "This is very very long response from payment gateway."
```

Comment:
If you need this, why not something like:
`"This is very very long response from payment gateway." * 8`